src/store/index.js

```diff
@@ -28,6 +29,19 @@ const store = createStore({
                     commit('setUser', {token: response.data.access_token, user: userData});
                     return response;
                 })
+        },
+        addMovie({commit}, movieData) {
+            return axios.post('/movie/add', movieData)
+                .then(response => {
+                    commit('addMovie', movieData)
+                    return response;
+                })
+        },
+        fetchMovies({commit}) {
+            axios.get('/movie/getall')
+                .then(response => {
+                    commit('setMovies', response.data)
+                })
```

Comment:
Nakon izmena na BE promenice ti se i rute koje gadjas. Popravi i to nakon BE izmena